prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.master.yaml

```diff
@@ -208,6 +208,24 @@ presubmits:
         nodeSelector:
           testing: test-pool
 
+    - name: release-e2e-stackdriver
```

Comment:
Can you add it as postsubmit first?

Reply:
We eventually want to block daily release if this test failed. I marked this test as optional so that it won't block now. Is there some reason that we should put it in postsubmit?

Reply:
When you add it as presubmit, even as optional, it is going to run for every PR immediately. I would like to see that the test is not flaky on postsubmit, before adding it to presubmit.

Reply:
I thought this is the release pipeline and it should only run with a new daily release?

Reply:
Any update?

Reply:
Oh :) You are right.